src/pages/javascript/es6/arrow_functions/index.md

```diff
@@ -37,4 +37,4 @@ let NewOneWithParameters = (a, b) => {
 NewOneWithParameters(10, 20);
 ```
 
-I don’t think I need to give explanation for this. It's straightforward.
+I don’t think, I need to give explanation for this. It's straightforward.
```

Comment:
Wouldn't the correct grammar be "I don't think I need to give **an** explanation for this"?

Reply:
Thank you. That is the correct version.